GPflowOpt/acquisition.py

```diff
@@ -456,3 +458,42 @@ def __mul__(self, other):
             return AcquisitionProduct(self.operands.sorted_params + other.operands.sorted_params)
         else:
             return AcquisitionProduct(self.operands.sorted_params + [other])
+
+
+class MCMCAcquistion(AcquisitionSum):
+    def __init__(self, acquisition, n_slices, **kwargs):
```

Comment:
The call to the parent constructor optimizes the first operand (which has restarts != 0), so it obtains the MLE.